File: src/BaseHmiTypes/Screens/Widgets/HmiScale.cs
using BaseHmiTypes.Screens.Base;

namespace BaseHmiTypes.Screens.Widgets;

/// <summary>
/// A static ruler-style scale with tick marks and optional labels.
/// </summary>
public class HmiScale : HmiScaleWidgetBase
{
    public HmiScale()
    {
        HmiObjectType = BaseHmiTypes.Screens.Base.HmiObjectType.HmiScale;
    }
}

File: typescript/base-hmi-types/src/screens/widgets/HmiScale.ts
import { HmiObjectType } from "../base/HmiObjectType.js";
import { HmiScaleWidgetBase } from "./HmiScaleWidgetBase.js";

/** A static ruler-style scale with tick marks and optional labels. */
export class HmiScale extends HmiScaleWidgetBase {
  constructor() {
    super();
    this.hmiObjectType = HmiObjectType.HmiScale;
  }
}
